recipes/views.py

```diff
@@ -24,7 +24,26 @@ def recipe_list(request):
 
 
 def recipe_detail(request, recipe_slug):
-    recipe = get_object_or_404(Recipe, slug=recipe_slug)
+    recipe_qs = (
+        Recipe.objects.prefetch_related("tags")
```

Comment:
amazing, pretty advanced orm querying here, how did you come up with it?

Reply:
gpt suggests to use `only` after filter to reduce prefetch work, e.g. `.only("id", "name", "slug")`

Reply:
Also good idea to add some indices to the db, checked with gpt and it suggested:

- Step: Index(["recipe", "order_id"])
- StepIngredient: Index(["step", "order_id"]), Index(["ingredient"])